proto/xudrpc.proto

```diff
@@ -26,7 +26,9 @@ service Xud {
     };
   }
   
-  /* Add an order to the order book. */
+  /* Add an order to the order book.
+   * If no price is specified a market order will get added
```

Comment:
Throw a period at the end here to be consistent with the other comments. Also I think it might make sense to say "If price is zero or unspecified a market order will get added" since explicitly setting a zero value for price will have the same effect as not setting it at all.